k9mail/src/test/java/com/fsck/k9/SputnikTest.java

```diff
@@ -0,0 +1,9 @@
+package com.fsck.k9;
+
+import java.util.*;
```

Comment:
[Checkstyle] INFO: Using the '.*' form of import should be avoided - java.util.*.